### PR TITLE
[IL-58] [QA] Cover products API functionality with autotests

### PR DESCRIPTION
**Pull request by Sanya124**
Task's name:
[IL-58] [QA] Cover products API functionality with autotests

Developer's full name:
Aleksandr Tolstikhin

Link to Documentation:
https://github.com/Sunagatov/Online-Store/wiki/API-Specification-for-Product

Changes:
- Tests have been developed for endpoints/api/v1/products. Also added asserts for checkings and queries to DB.
- Postgres Client Implementation.
- The steps for running tests in the README.md are described.

### CODE REVIEW

**Review by vitaliibredun — APPROVED** [review]
Approved

File: framework/clients/__init__.py
Comment (shpali4) [inline framework/clients/__init__.py:1]: why you need empty file? maybe you move some info inside?

File: framework/endpoints/__init__.py
Comment (shpali4) [inline framework/endpoints/__init__.py:1]: same

File: framework/queries/__init__.py
Comment (shpali4) [inline framework/queries/__init__.py:1]: same

File: pytest.ini
Comment (shpali4) [inline pytest.ini:1]: same

File: tests/product/__init__.py
Comment (shpali4) [inline tests/product/__init__.py:1]: same

File: tests/product/test_all_products.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import pytest
+
+from allure import description, feature, link, step, title
+from hamcrest import assert_that, is_
+from random import randint
+
+from framework.asserts.product_asserts import check_mapping_db_to_api
+from framework.endpoints.product_api import ProductAPI
+
```
Comment (shpali4) [inline tests/product/test_all_products.py:9]: delete this empty row pls

File: tests/product/test_product.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import uuid
+
+from allure import description, feature, link, step, title
+from hamcrest import assert_that, is_
+
+from framework.asserts.product_asserts import check_mapping_db_to_api
+from framework.endpoints.product_api import ProductAPI
+
```
Comment (shpali4) [inline tests/product/test_product.py:8]: same

**Review by Sunagatov — APPROVED** [review]

**Review by AiratGimaev — COMMENTED** [review]
Approved

**Pull request merged**